### DISCUSSION TITLE
Coqui do Indonesian text to speech

### DISCUSSION THREAD

**Discussion by SyamsQ** [coqui-ai/TTS#3803] [General Q&A]
Can Coqui do Indonesian text to speech?

I downloaded Coqui, but there is no Indonesian language there.

Please give me some guidance.

**Comment by eginhard** [comment]
The Fairseq models include Indonesian and and can be used with Coqui: https://github.com/idiap/coqui-ai-TTS?tab=readme-ov-file#example-text-to-speech-using-fairseq-models-in-1100-languages-

**Reply by SyamsQ** [reply]
Why, only these languanges? How to add Indonesian?
![image](https://github.com/user-attachments/assets/2c1b75cf-238e-4768-906f-4a010e62e68e)
